README.md

```diff
@@ -116,3 +116,28 @@ The OpenShift descriptor will automatically create `Route`s for Prometheus and J
    > For production-ready Jaeger instances, please see the [Jaeger Operator documentation](https://operatorhub.io/operator/jaeger) for how to properly deploy and configure production-ready instances.
    >
    > For production-ready OpenTelemetry Collector instances, please see the [OpenTelemetry Operator documentation](https://operatorhub.io/operator/opentelemetry-operator) for how to properly deploy and configure production-ready instances.
+
+### Jaeger
+
+Telemetry data is exported to a Jaeger instance, which can be analyzed and visualized at `http://localhost:16686`. 
+By now you've performed a few battles, so let's analyze the telemetry data.
+Open the Jaeger UI by going to `http://localhost:16686`.
```

Comment:
This won't necessarily be true unless they are running everything locally. We mention above (in both the docker compose section & the deploying to kubernetes section) that Jaeger is deployed and how to access it.

Maybe we just say "Open the Jaeger UI based on how you are running the system, either through [Docker Compose](#running-locally-via-docker-compose) or by [deploying the monitoring stack to kubernetes](#monitoring).